iotdb/src/main/java/org/apache/iotdb/db/writelog/replay/ConcreteLogReplayer.java

```diff
@@ -57,7 +57,7 @@ public void replay(PhysicalPlan plan, boolean isOverflow) throws ProcessorExcept
       }
     } catch (Exception e) {
       throw new ProcessorException(
-          String.format("Cannot replay log %s, because %s", plan.toString(), e.getMessage()));
+          String.format("Cannot replay log %s, because %s", plan.toString(), e.getMessage()), e);
```

Comment:
I am not sure if the e.getMessage() is redundant here.

Reply:
Removed.